### PR TITLE
feat: #101 Create All Ways Page

### PR DESCRIPTION
**Pull request by Lelik7777**

### CODE REVIEW

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styles from "src/pages/pageAllWays/AllWaysPage.module.scss";
+/**
```
Comment (tritonJS826) [inline mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:2]: add empty line, adjust linter

File: mw-webapp/src/router/pages.tsx [hunk 2]
```diff
@@ -18,4 +19,8 @@ export const pages = {
     path: "*",
     element: <Page404 />,
   },
+  waysPage: {
+    path: "waysPage",
+    element: <AllWaysPage />,
+  },
```
Comment (tritonJS826) [inline mw-webapp/src/router/pages.tsx:25]: put it above path *

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/package-lock.json
Comment (Ekaterina1994) [inline mw-webapp/package-lock.json:1]: Do we really need this file?

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import styles from "src/pages/pageAllWays/AllWaysPage.module.scss";
```
Comment (Ekaterina1994) [inline mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:1]: why the name of page is not WaysPage?

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+/**
+ * Page contains ways to all pages
```
Comment (Ekaterina1994) [inline mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:3]: It's wrong comment/. It is not page for all pages...

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+ */
+export const AllWaysPage = () => {
```
Comment (Ekaterina1994) [inline mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:5]: Rename

File: mw-webapp/src/router/Router.tsx [hunk 6]
```diff
@@ -16,6 +16,10 @@ export const Router = () => {
         path={pages.page404.path}
         element={pages.page404.element}
       />
+      <Route
+        path={pages.waysPage.path}
+        element={pages.waysPage.element}
+      />
```
Comment (Ekaterina1994) [inline mw-webapp/src/router/Router.tsx:22]: Move before 404 route

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/button/Button.tsx [hunk 7]
```diff
@@ -8,6 +8,7 @@ interface ButtonProps {
   /**
   * Callback triggered on button click
   */
+
```
[the remark this answers is not in the document]
Comment (Lelik7777) [reply mw-webapp/src/component/button/Button.tsx:11]: done

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+  return (
+    <div className={styles.main}>
```
[the remark this answers is not in the document]
Comment (Lelik7777) [reply mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:7]: done

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx (hunk 1 above)
Comment (Lelik7777) [reply mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:2]: done

File: mw-webapp/src/router/pages.tsx (hunk 2 above)
Comment (Lelik7777) [reply mw-webapp/src/router/pages.tsx:25]: done

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx (hunk 3 above)
Comment (Lelik7777) [reply mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:1]: done

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx (hunk 5 above)
Comment (Lelik7777) [reply mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:5]: done

File: mw-webapp/src/router/Router.tsx (hunk 6 above)
Comment (Lelik7777) [reply mw-webapp/src/router/Router.tsx:22]: done

File: mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx (hunk 4 above)
Comment (Lelik7777) [reply mw-webapp/src/pages/pageAllWays/AllWaysPage.tsx:3]: done

**Comment by Shustovsky** [comment]
please delete package-lock.json

File: mw-webapp/package-lock.json
Comment (Lelik7777) [reply mw-webapp/package-lock.json:1]: done

**Review by Shustovsky — APPROVED** [review]

**Review by chelovek-zeml — APPROVED** [review]

File: mw-webapp/src/router/Router.tsx (hunk 6 above)
Comment (Howl404) [reply mw-webapp/src/router/Router.tsx:22]: You forgot to push comments? It still after page 404

File: mw-webapp/src/pages/waysPage/WaysPage.module.scss [hunk 9]
```diff
@@ -0,0 +1,5 @@
+.container{
```
Comment (Howl404) [inline mw-webapp/src/pages/waysPage/WaysPage.module.scss:1]:
```suggestion
.container {
```

**Review by Howl404 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/pages/waysPage/WaysPage.module.scss (hunk 9 above)
Comment (Lelik7777) [reply mw-webapp/src/pages/waysPage/WaysPage.module.scss]: done

**Review by Arterixs — APPROVED** [review]

File: mw-webapp/pnpm-lock.yaml
Comment (Howl404) [inline mw-webapp/pnpm-lock.yaml:1]: Seems like bad pnpm lock merge, <<<<< HEAD from merge, maybe run pnpm install again

**Review by Howl404 — DISMISSED** [review]

File: mw-webapp/pnpm-lock.yaml
Comment (Lelik7777) [reply mw-webapp/pnpm-lock.yaml:1]: done

**Review by SergioAJS — APPROVED** [review]

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]
Check PR name and docs

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**